runners/katacoda/index.ts

```diff
@@ -404,30 +404,25 @@ export class Katacoda extends Runner {
 
     runExecuteCommand(runCommand: RunCommand) : RunResult {
         let terminal = (runCommand.command.parameters.length > 2 && runCommand.command.parameters[2].asynchronous) 
-            ? this.getTerminal("executeCommand"+runCommand.stepIndex) 
+            ? this.getTerminal("executeCommand" + runCommand.stepIndex)
             : undefined;
-        
-        let filepath;
-        let changeDir = false;
+
+        let cdCommand: string = undefined; 
```

Comment:
I think that you need to add this.changeCurrentDir(path.join(this.getVariable(this.WORKSPACE_DIRECTORY)) here